src/test/java/io/jenkins/plugins/checks/steps/PublishChecksStepTest.java

```diff
@@ -43,19 +49,53 @@ void shouldPublishCheckWithDefaultValues() throws IOException, InterruptedExcept
     }
 
     @Test
-    void shouldPublishCheckWithSetValues() throws IOException, InterruptedException {
-        PublishChecksStep step = new PublishChecksStep();
-        step.setName("Jenkins");
-        step.setSummary("a check made by Jenkins");
-        step.setTitle("Jenkins Build");
-        step.setText("a failed build");
-        step.setStatus(ChecksStatus.IN_PROGRESS);
-        step.setConclusion(ChecksConclusion.FAILURE);
-        step.setDetailsURL("http://ci.jenkins.io");
+    void shouldPublishCheckWithStatusInProgress() throws IOException, InterruptedException {
+        PublishChecksStep step = getModifiedPublishChecksStepObject("an in progress build",
+                ChecksStatus.IN_PROGRESS,null);
```

Comment:
```suggestion
                ChecksStatus.IN_PROGRESS, null);
```

Same for the next check style warning